### DISCUSSION TITLE
How to make RTM in a separate window and make this windows always at the top?

### DISCUSSION THREAD

**Discussion by luckcao** [netless-io/flat#1594] [Q&A]
Hello, 
I'm using the newest version of Flat(Windows 10 OS) and I have a question:
How to make RTM in a separate window and make this windows always on the top?

Because, after I shared my desktop to other people and opened other program(window) in maximized mode, then, I cann't see the message that sent by other people. If I want to see the message, I need make Flat program windows as active window. It's not a good feeling of using.

Maybe there is a way to set this, but, I didn't find.
Could anyone help me?

Thanks a lot!

**Comment by BlackHole1** [comment]
Cool! This is a good idea and I approve of this feature. @Leooeloel  Do you have any thoughts on this?

**Comment by Leooeloel** [comment]
Thanks for the feedback, this is a great idea, we will optimize.
